server_model.js

```diff
@@ -99,6 +99,9 @@ Asana.ServerModel = {
         "GET", "/workspaces/" + workspace_id + "/users",
         { opt_fields: "name,photo.image_60x60" },
         function(response) {
+          for (user in response) {
```

Comment:
Hmm,  isn't the response an *array* of dictionaries? Not sure this will do what you want.
Also, please generally avoid the `for .. in` construct. Historically it has been non-performant. In this particular codebase it's probably fine, but just be aware there are lots of "better" ways of iterating keys in a dictionary if that's what you want to do.